model/Software/Classes/Package.md

```diff
@@ -40,7 +40,7 @@ Note that some of these could be represented in SPDX as a file as well.
   - type: xsd:anyURI
   - minCount: 0
   - maxCount: 1
-- homePage
+- doap:homepage
```

Comment:
do we have a place, where we fix what `doap` expands to?

Reply:
yes

Reply:
Yes - there is a file defined where all other namespaces (e.g. XSD) - we will add DOAP to this